pkgs/os-specific/linux/disk-indicator/default.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, fetchgit, libX11 }:
 
 stdenv.mkDerivation {
-  name = "disk-indicator";
+  name = "disk-indicator-git-2014-05-19";
```

Comment:
It affects the package name as understood by `nix-env`, which means that you'll need to do `nix-env -i disk-indicator-git` and an "upgrade" will no longer pick this up as a new version.